test/karma.conf.js

```diff
@@ -56,6 +56,7 @@ module.exports = function setKarmaConfig(config) {
           'process.env': {
             NODE_ENV: JSON.stringify('test'),
             CI: JSON.stringify(process.env.CI),
+            KARMA: JSON.stringify(true),
```

Comment:
JSON.stringify(true) => "true" same for "test"

Reply:
I'm not familiar with how the value gets converted. The relevant parts of the logic:

- https://github.com/mui-org/material-ui/blob/06ac77b52cdda7e3bcc0fb1dfaf3de25e287854e/test/utils/mochaHooks.js#L3
- https://github.com/mui-org/material-ui/blob/c500fc3faf873142b9124319cebf728dd9a12cc5/test/karma.conf.js#L62